app/commands/user/combine_authors_and_contributors.rb

```diff
@@ -0,0 +1,11 @@
+class User::CombineAuthorsAndContributors
```

Comment:
The doesn't feel like a command to me as it doesn't actually do anything mutating, and it doesn't feel like it's part of the `User` scope. I'll have a think about where it should go.

Reply:
Yeah I know. Maybe an assembler?